src/lwc/geModalPrompt/geModalPrompt.js

```diff
@@ -7,7 +7,19 @@ export default class geModalPrompt extends LightningElement {
     @api variant = '';
     @api title;
     @api message;
-    @api buttonText;
+    @api buttons = [];
+
+    @track enrichedButtons = [];
+
+    connectedCallback() {
```

Comment:
Would it be possible to pass in the result of "enrichedButtons" through the buttons api property? So, enrich them in a parent component and then just pass them to this component? 

Not saying that this _has_ to change, but could the use of connectedCallback possibly be avoided that way?

Reply:
Yes, it definitely could! The intent here was to define a default button behavior of "modal close", just to make the geModalPrompt component a bit more light weight in the simplest use case - single "okay" button that just dismisses the modal. This allows you to define a more basic prompt modal without having to pass in your button action, like in like 447 of the form app.

However, I was honestly on the fence about this - if you have a preference to pass in the actions in all cases and remove this connected callback, that would nudge me in the direction of that and it would be a small enough change to make.

Reply:
I see, yeah if its not too big of a lift, I think we should avoid use of the connectedCallback.  Maybe the idea of a simple default button could be added later as a slot or something...